uniffi_bindgen/src/bindings/swift/gen_swift/miscellany.rs

```diff
@@ -22,6 +22,10 @@ macro_rules! impl_code_type_for_miscellany {
                      $class_name.into()
                  }
 
+                 fn canonical_name(&self, _oracle: &dyn CodeOracle) -> String {
+                    $canonical_name.into()
+                }
+
```

Comment:
I've also checked Python which has an implementation already and Ruby which hasn't moved to the unit-o'-code pattern.